annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -0,0 +1,145 @@
+package edu.isi.vista.annotationutils
+
+import com.jcraft.jsch.JSch
+import com.jcraft.jsch.Session
+import edu.isi.nlp.parameters.serifstyle.SerifStyleParameterFileLoader
+import java.io.File
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.api.ResetCommand
+import org.eclipse.jgit.api.TransportConfigCallback
+import org.eclipse.jgit.diff.DiffEntry
+import org.eclipse.jgit.lib.ObjectId
+import org.eclipse.jgit.revwalk.RevCommit
+import org.eclipse.jgit.revwalk.RevWalk
+import org.eclipse.jgit.transport.*
+import org.eclipse.jgit.treewalk.CanonicalTreeParser
+import org.eclipse.jgit.util.FS
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
```

Comment:
💯 for a good "how to use this" comment.

I suggest shifting the `main` method from the bottom of the file to the top and making this the docstring for that method.  (Generally, it is good to order the classes, functions, etc. in a source file in the order a user trying to understand the program should ready them. You have more freedom to do this in Java/Kotlin/etc. than in Python, which has some ordering constraints)

Reply:
Moved `main` to top of file and used comment as docstring

Reply:
Almost! To make your comment the docstring for `main`, [you want to put it immediately before the function](https://kotlinlang.org/docs/reference/kotlin-doc.html) rather than just inside it (like Python doeS)

Reply:
Done - same for `setUpRepository`